Project-2/README.md

```diff
@@ -10,42 +10,261 @@ Tip #4: Work together as a team.
+
+ΔI2 = {Plants, Cryptogams, Phanerograms}
+
+Role Assignments:
+s2 = {(Plants, Cryptogams),  (Plants, Phanerograms)}
+
+Bisimulation:
+
+ρ = {(Animals, Plants), (Vertebrates, Cryptogams), (Invertebrates, Phanerograms)}
+
+So Animals is bisimilar to Plants.
+
 
 6. Explain the difference - using natural language - between the description logic expressions:
-  ```
+ 
   (a) ∃r.C and ∀r.C
+  
+  Answer: ∃r.C means that all x has a r-filler y which belongs to C, while ∀r.C means that all r-fillers ys of all x belong to C.
+  
   (b) ∃r-.C and ∀r-.C
+  
+  Answer: ∃r-.C means that there is a thing y r-related to all xs, and this thing belongs to C,
+  
+  while ∀r.C means that if there is a thing r-related to all xs, then this thing belongs to C.
+  
   (c) <=nr and <=nr.C
+  
+  Answer: <=nr means that role r connects all the xs to no more than n elements.
+ 
+  while <=nr.C means that role r connects all the xs to no more than n elements, and they belong to C.
+  
   (d) ∃r-.C and ∃r-.{a} 
-```
+  
+  Answer: ∃r-.C means that for all xs, there is at least a thing y, which is r-inverse related to it, and which belongs to C,
+  
+  while ∃r-.{a} means that for all xs, there is element a, which is r-inverse related to it.
+
 
 7. There is a delightfully helpful subreddit called "ELI5" which stands for something like "explain it like I'm 5" where users post conceptually challenging questions and other users attempt to provide explanations in simple, jargon-free, terms that presumably a 5 year-old could understand. Using this as a model, explain the _finite model property_. Be sure to provide a simple example and explain when the property might be important, and when it is not so important. 
 
+Ansewr: 
+Bob loves playing games, and he hopes there is a game that can be played with different results forever. 
+One day, his father gives him a game, called Color-card: there are a number of color cards in a box. Each card has a different color, but it is full colored on one side, while on the other side it is a blank sheet with the colored dot. Once a time, the player chooses any number of cards from the box, and then use any number of cards he chose to make combinations. For example, the first time, Bob chooses five color cards from the box, red, yellow, blue, green, brown, then he uses three cards, and put them in such a order: 1, red (front side), 2, blue (back side), 3, green (back side). That is one combination. The player continues playing until there is a combination that repeats. If the player cannot continue playing, he/she fails. 
+However, if the player fails at one time, he/she can choose more or/and different cards, then play the next time. The player wins if the cards he chooses can allow him to play forever. 
+Unfortunately, Bob cannot win the game, because no matter how many cards he chooses (the number of cards is not limitless), and how hard he tries, he cannot continue playing without a repeated combination occurs.  
```

Comment:
I like this story, but it's not clear to me how it elucidates the finite model property, other than by saying this card game has it. Examples of formal structures that have the property are of course helpful when ELI5, but we should include a description of the features of the example that are generalizable wrt finite model property.